index.html

```diff
@@ -0,0 +1,127 @@
+<!doctype html>
+<html class="no-js" lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title> Delicious | THE BEST FOOD BLOG ON THE WEB </title>
+    <meta name="description" content="Delicious: the best food blog on the web">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="apple-touch-icon" href="apple-touch-icon.png">
+    <!-- Place favicon.ico in the root directory -->
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/skeleton.css">
+    <link rel="stylesheet" href="css/main.css">
+    <link rel="stylesheet" href="css/icomoon.css">
+    <link href='http://fonts.googleapis.com/css?family=Halant:300,400|Qwigley|Yanone+Kaffeesatz:400,300|Roboto:300,400' rel='stylesheet' type='text/css'>
+    <link href='http://fonts.googleapis.com/css?family=Coming+Soon' rel='stylesheet' type='text/css'>
+    <script src="js/vendor/modernizr-2.8.3.min.js"></script>
+  </head>
+  <body>
+    <div class="container">
+      <header class="banner">
+        <div class="row">
+          <div class="eight columns">
+            <section class="logo">
+              <img src="http://i.imgur.com/5tFwpoN.png">
+              <section class="logo-text">
+                <h1> Delicious </h1>
+                <h2> THE BEST FOOD BLOG ON THE WEB </h2>
+            </section>
+          </div>
+          <div class="four columns">
+          <nav class="social-icons">
+              <ul>
+                <li><span class="icon-facebook2"></li>
+                <li><span class="icon-twitter2"></li>
+                <li><span class="icon-google-plus2"></li>
+                <li><span class="icon-camera"></li>
+                <li><span class="icon-flickr3"></li>
+                <li><span class="icon-pinterest"></li>
+                <li><span class="icon-station2"></li>
+                <li><span class="icon-mail3"></li>
+              <ul>
+            </nav>
+          </div>
+        </div>
+      </header>
+      <main>
+        <article>
+          <header class="recipe-header">
+            <div class="row">
+              <div class="ten columns">
+                <h1> Chocolate Pizza </h1>
+                <p> POSTED ON <time datetime="2013-12-15"> 15 DEC 2013 </time> / DESSERTS </p>
+              </div>
+              <div class="two columns">
+                <section class="print">
+                  <p><span class="icon-printer2"> PRINT</p>
+                </section>
+              </div>
+            </div>
+            <img class="food-pic" src="http://i.imgur.com/fB4svJW.png">
```

Comment:
This could be taken out of the 'header' if it's getting in the way of the 'row'. Also, don't forget alt text, again